base/Event.js

```diff
@@ -0,0 +1,11 @@
+class Event {
+  constructor(client, {
+    name = null,
+    location = ""
```

Comment:
I'm not sure why the location would be required as you don't get "sub-events" like you would do with sub-commands.

Reply:
Right now location is required due to how they're reloaded. If you look at the `reloadEvent` function, it needs some way to require the file. I could just hardcode the events folder location in if that's preferred.

Reply:
The original event loader was straight up hardcoded to just load the `./events/` directory, not sure why that behaviour would warrant changes as previously mentioned there isn't any reason for sub-folders within the events directory.

Reply:
Yeah, changed in upcoming commit.